.eslintignore

```diff
@@ -0,0 +1,26 @@
+/docs
+/mocks
+/node_modules
+/public
+/scripts
+/spec
+
+/src/api
```

Comment:
Folders placed in `/src` should not be ignored

Reply:
for now only /src/redux is under lint. Only new code will be put to lint, no sense to lint old code, it will redo to different structure